### DISCUSSION TITLE
What happened to Custom Colors options?

### DISCUSSION THREAD

**Discussion by Nano-Ocelot** [darkreader/darkreader#10231] [Q&A] [answered]
I used to use a feature in the dropdown ui that let me choose custom colors to convert webpages to. For some reason those colors are suddenly not working anymore (the webpages are a default gray again) and I see no option to change it back. Was this feature completely removed? I really hope not. That was my main use for this extension. 

Edit: Upon further digging the entire UI looks like it looked over 2 years ago. Was there some massive regression? It appears to be missing lots of features now. I am on Firefox btw.

**Comment by Gusted** [comment, marked as answer]
Hi @Nano-Ocelot,

It seems like you switched to the old UI, you can move back to the new UI:
- Click Dark Reader icon.
- Click Dev tools (in the bottom-right corner).
- Click Preview new design.

**Reply by Nano-Ocelot** [reply]
Wow, thank you. No idea how that happened. Problem solved.
